requirements/edx/kernel.in

```diff
@@ -114,7 +114,8 @@ olxcleaner
 openedx-calc                        # Library supporting mathematical calculations for Open edX
 openedx-django-require
 # openedx-events 3.1.0 introduces producer API
-openedx-events>=3.1.0               # Open edX Events from Hooks Extension Framework (OEP-50)
+# openedx-events>=3.1.0               # Open edX Events from Hooks Extension Framework (OEP-50)
+openedx-events @ git+https://github.com/open-craft/openedx-events.git@4d2d951944f72c622e1bed909ce13941b096f913
```

Comment:
**TO DO** So we don't forget: this requirement needs to be updated when https://github.com/openedx/openedx-events/pull/244 merges:
```suggestion
openedx-events>=8.3.0               # Open edX Events from Hooks Extension Framework (OEP-50)
```